src/polyswarm/utils.py

```diff
@@ -95,4 +97,34 @@ def is_valid_id(value):
         int(value)
         return True
     except:
-        return False
\ No newline at end of file
+        return False
+
+
+#
+# URL validations, port from the same validations applied by Portal
+#
+
+URL_REGEX = re.compile(
```

Comment:
Hey guys, URL validation is a pretty comples problem. For reference:

https://stackoverflow.com/questions/161738/what-is-the-best-regular-expression-to-check-if-a-string-is-a-valid-url?page=1&tab=scoredesc#tab-top

https://mathiasbynens.be/demo/url-regex

Are we sure we are not blocking things here that we would like to have scanned by the sandbox accidentally? Where did these come from? Are they compliant with some RFC?

Reply:
We have had problems in the past with the regex in the Frontend, and it has been curated over time, so we are pretty confident it work OK.

That said, I can change the algorithm to the following:

- If URL starts with "http://" or "https://" that's enough.
- Otherwise IP validation or domain regex are applied.
- If no one of the validations above pass, it's not a valid URL.

Reply:
Implemented what I proposed in the last comment:


> - If URL starts with "http://" or "https://" that's enough.
> - Otherwise IP validation or domain regex are applied.
> - If no one of the validations above pass, it's not a valid URL.